librz/include/rz_th.h

```diff
@@ -74,25 +74,27 @@ typedef struct rz_th_t {
 	RzThreadLock *lock;
 	RZ_TH_FUNCTION(fun);
 	void *user; // user pointer
-	int running;
-	int breaked; // thread aims to be interrupted
+	bool running;
+	bool breaked; // thread aims to be interrupted
 	int delay; // delay the startup of the thread N seconds
-	int ready; // thread is properly setup
+	bool ready; // thread is properly setup
```

Comment:
As a sidenote (not directly related to your additions), a lot of these members should be removed, especially `ready`, `delay` and `breaked` and `lock`. `RzThread` should be a pure portability and provide approximately the same interface as pthreads, but not more. Anything else can be built around.

Reply:
i agree. that requires a bit more of work